Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelemetryOperations.java

```diff
@@ -0,0 +1,39 @@
+package tddmicroexercises.telemetrysystem;
+
+public class TelemetryOperations implements Operations {
+
+
+    private final String DiagnosticChannelConnectionString;
+
+    DiagnosticControls telemetryDiagnostic = new TelementryDiagnosticControlss();
```

Comment:
I also meant to pass TelementryDiagnosticControlss object in Constructor param so we can remove hard Coupling (try not to use "new" keyword as much as possible)